packages/block-library/src/heading/style.native.scss

```diff
@@ -2,3 +2,6 @@
 .blockText {
 	min-height: $min-height-heading;
 }
+.block-library-heading__heading-level-checker {
```

Comment:
This style if really necessary should be applied to `style.scss` file which web version uses. This file is used by React Native app.